### PR TITLE
Ruff & project standardization

### PR DESCRIPTION
**Pull request by bandophahita**
implementing ruff and project standardization
https://github.com/ScreenPyHQ/screenpy/issues/127

### CODE REVIEW

File: screenpy_pyotp/__version__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-                      ╔═╝║        ╔═╝║
-                      ╚══╝        ╚══╝
+                ╔═══╗             ╔═══╗       ╔═══╗   ╔═══╦════╦═══╗
+                ║╔═╗║             ║╔═╗║       ║╔═╗║   ║╔═╗║╔╗╔╗║╔═╗║
+                ║╚══╦══╦═╦══╦══╦═╗║╚═╝╠╗ ╔╗   ║╚═╝╠╗ ╔╣║ ║╠╝║║╚╣╚═╝║
+                ╚══╗║╔═╣╔╣║═╣║═╣╔╗╣╔══╣║ ║║   ║╔══╣║ ║║║ ║║ ║║ ║╔══╝
+                ║╚═╝║╚═╣║║║═╣║═╣║║║║  ║╚═╝║   ║║  ║╚═╝║╚═╝║ ║║ ║║
+                ╚═══╩══╩╝╚══╩══╩╝╚╩╝  ╚═╗╔╝   ╚╝  ╚═╗╔╩═══╝ ╚╝ ╚╝
+                                      ╔═╝║        ╔═╝║
+                                      ╚══╝        ╚══╝
```
Comment (perrygoy) [inline screenpy_pyotp/__version__.py:9]: 🤌 💋

File: setup.py
Comment (perrygoy) [inline setup.py:1]: Should we be deleting `setup.py` as part of our project standardization?

File: tests/test_abilities.py [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
 
 class TestAuthenticateWith2FA:
-    def test_can_be_instantiated(self):
-        a1 = AuthenticateWith2FA.using_secret("")
-        a2 = AuthenticateWith2FA.using(None)
+    def test_can_be_instantiated(self, mocked_pyotp: mock.Mock) -> None:
+        a1 = AuthenticateWith2FA(mocked_pyotp)
+        a2 = AuthenticateWith2FA.using_secret("")
+        a3 = AuthenticateWith2FA.using(mocked_pyotp)
```
Comment (perrygoy) [inline tests/test_abilities.py:12]: Nice catch on this one.

**Review by perrygoy — DISMISSED** [review]
Looks good! I only had one comment about removing `setup.py` that we might want to consider.

**Review by perrygoy — APPROVED** [review]
Looks good! We'll remove `setup.py` in a future one.

File: setup.py
Comment (bandophahita) [reply setup.py:1]: I think it should be done in a separate PR.

**Pull request merged**